src/components/dashboard/SendQRSummary.js

```diff
@@ -139,17 +141,15 @@ const SendQRSummary = ({ screenProps }: AmountProps) => {
           <Section.Stack grow={3}>
             <CustomButton
               mode="contained"
-              onPress={async () => {
-                ;(await goodWallet.isCitizen()) ? sendGD() : faceRecognition()
-              }}
+              onPress={() => (Config.isEToro ? setShowSurvey(true) : confirm())}
               loading={loading}
             >
               Confirm
             </CustomButton>
           </Section.Stack>
         </Section.Row>
       </Section>
-      <SurveySend handleCheckSurvey={setSurvey} />
+      {showSurvey && <SurveySend handleCheckSurvey={setSurvey} onDismiss={confirm} />}
```

Comment:
does onDismiss happens also when user presses the X and not OK?

Reply:
yes in both cases